### ISSUE TITLE
If the user organisation number is exactly 1 `PickOrganisation` doesn't let the user choose himself as an organisation

### ISSUE THREAD

**Issue by jody-frankowski** [jenkins-x/jx#450]
If I understand this right, the function `PickOrganisation` should let the user choose which organisation he wishes to create the repo in when jx creates one for him.

[Relevant code](https://github.com/jenkins-x/jx/blob/279d28405eb63436ca7e871f7814f530497f07f1/pkg/gits/provider.go#L225)

I guess it's just a matter of removing the `len(orgs) == 1` check, but I might be wrong :)

I have been bitten by this when I did `jx create spring`:
```sh
[...]
Git repository created
replacing placeholders in direcory /home/me/test/jx/demo
app name: demo, git server: github.com, org: jody-frankowski
[...]
Creating repository ORG/jx-test
[...]
```

**Comment by jlordiales** [comment]
This should have been fixed with #486 that was merged today hopefully

**Comment by jstrachan** [comment]
thanks @jlordiales yeah there's a new binary out with this fix: https://github.com/jenkins-x/jx/releases I wonder @jody-frankowski if you could double check if this is now fixed for you?
